subsys/bluetooth/host/hci_core.h

```diff
@@ -202,6 +202,9 @@ struct bt_le_per_adv_sync {
 	uint8_t cte_type;
 #endif /* CONFIG_BT_DF_CONNECTIONLESS_CTE_RX */
 
+	/** Reassembly buffer for advertising reports */
+	struct net_buf *reassembly_buffer;
```

Comment:
That's a pretty long name. Maybe `reassembly_buf`, or perhaps even better just `reassembly`? I don't think that hurts readability of the code in any way.